### PR TITLE
AVM migration module docs updates

### PR DESCRIPTION
**Pull request by jtracey93**
# Overview/Summary

AVM migration module docs updates

## This PR fixes/adds/changes/removes

1. AVM migration module docs updates

### Breaking Changes

None. Docs only

## Testing Evidence

N/A

## As part of this Pull Request I have

- [ ] Read the [Contribution Guide](https://github.com/Azure/bicep-lz-vending/wiki/contributing) and ensured this PR is compliant with the guide
- [ ] Checked for duplicate [Pull Requests](https://github.com/Azure/bicep-lz-vending/pulls)
- [ ] Associated it with relevant [GitHub Issues](https://github.com/Azure/bicep-lz-vending/issues)
- [ ] *(LZ-Vending Core Team Only)* Associated it with relevant [ADO Items](https://aka.ms/lz-vending/backlog)
- [ ] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/bicep-lz-vending/tree/main)
- [ ] Performed testing and provided evidence.
- [ ] Updated one or more of the following tests *(if required)*
  - [Unit](https://github.com/Azure/bicep-lz-vending/blob/main/.github/workflows/bicep-build-to-validate.yml)
  - [Linting](https://github.com/Azure/bicep-lz-vending/tree/main/.github/workflows/code-review.yml)
- [ ] Updated relevant and associated documentation (e.g. Contribution Guide, Module READMEs, Wiki Docs etc.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,11 +3,49 @@
 [![Average time to resolve an issue](http://isitmaintained.com/badge/resolution/azure/bicep-lz-vending.svg)](http://isitmaintained.com/project/azure/bicep-lz-vending "Average time to resolve an issue")
 [![Percentage of issues still open](http://isitmaintained.com/badge/open/azure/bicep-lz-vending.svg)](http://isitmaintained.com/project/azure/bicep-lz-vending "Percentage of issues still open")
 
+_____________________________________________________________________________________________________________________________________
+
+## ⚠️⚠️ Repository archival notice ⚠️⚠️
+
+We would like to inform you that this repository has been **archived** and the Bicep landing zone vending module for Azure now has a new home in the [Bicep public registry](https://aka.ms/lz-vending/bicep).
```
Comment (jtracey93) [inline README.md:10]:
```suggestion
We would like to inform you that this repository has been **archived** and the Bicep landing zone vending module for Azure now has a new home in the [Bicep public registry](https://aka.ms/lz-vending/bicep) and is now a pattern module in the [Azure Verified Modules](https://aka.ms/avm) initiative.
```

File: README.md [hunk 2]
```diff
@@ -3,11 +3,49 @@
...
+
+### What does this mean?
+
+- This repository is now in a **read-only** state.
+- No further issues, pull requests, or updates will be maintained on this repository.
+- All future releases, fixes and improvements will be managed in the new module hosted on the [Bicep public registry](https://aka.ms/lz-vending/bicep).
```
Comment (jtracey93) [inline README.md:16]:
```suggestion
- All future releases, fixes and improvements will be managed in the new AVM pattern module hosted on the [Bicep public registry](https://aka.ms/lz-vending/bicep).
```

File: README.md [hunk 3]
```diff
@@ -3,11 +3,49 @@
...
+- We will continue to leverage the [Wiki](https://github.com/azure/bicep-lz-vending/wiki) in this repository as the module's documentation and guidance location.
```
Comment (jtracey93) [inline README.md:17]:
```suggestion
- We will continue to leverage the [Wiki](https://github.com/azure/bicep-lz-vending/wiki) in this repository as the module's additional documentation and guidance location.
```

File: README.md [hunk 4]
```diff
@@ -3,11 +3,49 @@
...
+### Why was this done?
+
+We are converting this module to be an [Azure Verified pattern module](https://aka.ms/avm) to better align with the [Azure Verified Modules](https://azure.github.io/Azure-Verified-Modules/concepts/what-why-how/) initiative. This will help us align with the Well-architected framework guidance, have improved [module support](https://azure.github.io/Azure-Verified-Modules/help-support/module-support/) and provide you with a consistent experience consuming [AVM Bicep modules](https://azure.github.io/Azure-Verified-Modules/indexes/bicep/).
+
+📒 If you are not familiar with [Azure Verified Modules](https://aka.ms/avm), you can watch the following Youtube videos for a quick overview:
+
+- [An Introduction to Azure Verified Modules (AVM) by the Customer Architecture & Engineering team](https://youtu.be/JbIMrJKW5N0?si=4v69hlyBbINufHEO) 📽️🎬🎞️
+- [Azure Verified Modules Overview by John Savill](https://youtu.be/3FeIFHaJOtg?si=ZDXJ3EPwT9Xlzq2P) 📽️🎬🎞️
+
+### If I'm already using the Bicep Subscription vending module from the Bicep public registry, what do I need to change?
```
Comment (jtracey93) [inline README.md:28]: lets add something here about `v1.5.2` becoming `v0.1.0` in AVM, or whatever the next release number would have been 👍

**Review by jtracey93 — COMMENTED** [review]
Looks good @sebassem please see comments for changes

**Review by sebassem — APPROVED** [review]
LGTM

**Review by sebassem — APPROVED** [review]

**Pull request merged**